drivers/adc/adc_mcux_lpadc.c

```diff
@@ -387,10 +390,22 @@ static int mcux_lpadc_init(const struct device *dev)
 	lpadc_config_t adc_config;
 	int err;
 
+#ifdef CONFIG_PINCTRL
```

Comment:
I believe `CONFIG_PINCTRL` was dropped already here by 989d103d5355c57d4533b78aa32a3033d33b142, as this code no longer needs to be conditional.

Reply:
ping here